omi/sawtooth_omi/handler.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import hashlib
+
+from sawtooth_sdk.processor.state import StateEntry
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.processor.exceptions import InternalError
+from sawtooth_sdk.protobuf.transaction_pb2 import TransactionHeader
+
+from sawtooth_omi.protobuf.work_pb2 import Work
+from sawtooth_omi.protobuf.recording_pb2 import Recording
+from sawtooth_omi.protobuf.identity_pb2 import IndividualIdentity, OrganizationalIdentity
+from sawtooth_omi.protobuf.txn_payload_pb2 import OMITransactionPayload
+
+
+# actions
+
+WORK = 'SetWork'
```

Comment:
Doesn't hurt to separate those out.  In fact, if you removed them, you probably don't need constants for the `SetX` actions.